cns/networkcontainers/networkcontainers.go

```diff
@@ -17,6 +17,20 @@ type NetworkContainers struct {
 	logpath string
 }
 
+// NetPluginConfiguration represent network plugin configuration that is used during Update operation
+type NetPluginConfiguration struct {
+	path              string
+	networkConfigPath string
+}
+
+// NewNetPluginConfiguration create a new netplugin configuration.
+func NewNetPluginConfiguration(binPath string, configPath string) (*NetPluginConfiguration, error) {
```

Comment:
It was needed because the object is defined in some other package (networkcontainers) and called from some where else (restserver). 
default constructor <struct-name> ( <member-variable> : value, <member-variable> : value)
I tried changing the visibility by changing the case, that didn't help either.